tools/generator/templates/champions/cxx/Makefile.jinja2

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+CHAMPION_SOURCES = champion.cc # Add other sources (*.cc) seperated by spaces
```

Comment:
To make it easy to modify I'd put the comments above the line, not in-line

Reply:
ok